TELCODOCS-429 removing TP from Pod-level bonding for secondary networks CNF-3877 (TELCODOCS-429)

kquinn1204 opened the pull request:
[TELCODOCS-429]: [CNF-3877](https://issues.redhat.com//browse/CNF-3877) Promote bond CNI to GA

Version(s):
4.12 RN

Issue:
https://issues.redhat.com/browse/TELCODOCS-429

see also PR https://github.com/openshift/openshift-docs/pull/47172 for context
Link to docs preview: http://file.emea.redhat.com/kquinn/TELCODOCS-429-410/release_notes/ocp-4-10-release-notes.html#ocp-4-10-technology-preview
<!--- Add direct link(s) to the exact page(s) with updated content from the preview build. --->

QE review:
- [ ] QE has approved this change.
<!--- QE approval is required to merge a PR except for changes that do not impact the meaning of the docs. --->

Additional information:
<!--- Optional: Include additional context or expand the description here.--->

<!--- After you open your PR, ask for review from the OpenShift docs team:
  For community authors: Tag @openshift/team-documentation in a GitHub comment.--->

kquinn1204 commented:
@mmirecki @cgoncalves I raised this to update the TP and remove the note "The current functionality of Bond CNI..." Franck asked me to update the docs as I assume fix filtered through in z stream . Can you give this a LGTM please

cgoncalves commented:
/lgtm

cgoncalves commented:
/lgtm

Release noting which z-stream feature support levels changed is definitely much better, thanks!

kalexand-rh commented:
/hold

kquinn1204 commented:
@kalexand-rh I have addressed your concerns about the footnote and removed for both pod level bonding and the Web Terminal Operator in this PR. I followed the azure example and put in a link to the correct errata description.

kquinn1204 commented:
@JoeAldinger Hopefully this can be merged

kquinn1204 commented:
/label merge-review-needed

jeana-redhat commented on release_notes/ocp-4-10-release-notes.adoc:
```diff
@@ -3369,6 +3360,14 @@ You can view the container images in this release by running the following comma
+[id="ocp-4-10-30-pod-level-bonding-ga"]
+===== General availability of pod-level bonding for secondary networks
+
+With this update, link:https://docs.openshift.com/container-platform/4.10/networking/hardware_networks/using-pod-level-bonding.html[*Using pod-level bonding*] is now generally available.
```
This should be an xref rather than a link.

jeana-redhat reviewed (commented):
This is very neatly done. The only issue I see is the link in the .z section
